forms-flow-web/src/components/Form/List.js

```diff
@@ -419,13 +419,33 @@ const List = React.memo((props) => {
   const fileUploaded = async (evt) => {
     FileService.uploadFile(evt, async (fileContent) => {
       let formToUpload;
-      if ("forms" in fileContent) {
+      //  to upload forms of a project from form.io
+      if ("roles" in fileContent) {
```

Comment:
As discussed Update the Changes removing the role check and forms check with form is Array or Object to know the context of forms import - forms key is an object when form.io export vs forms is an array in our case

Reply:
sure